### DISCUSSION TITLE
Switch screens

### DISCUSSION THREAD

**Discussion by tianhuahao** [chrxh/alien#91] [General]
How do I switch screens in this software? I have two monitors.

### Issue

Currently, using the `Win+Shift+→` keyboard shortcut to switch screens on Windows 10 only switches to the right screen temporarily. When I press the Win key to open another software, the Alien software will minimize and go back to the original screen, which is inconvenient.

### Request

Is there a way to permanently stay on the right screen? It would be great if there was a setting or workaround that would allow users to choose which screen they want to stay on without having to constantly switch back and forth.

Thank you in advance for your help!

**Comment by chrxh** [comment]
This feature is currently not implemented and will be added in the future (I also use 2 monitors). By now, ALIEN is stick to the primary monitor. If you want it permanently on the other monitor you could change the primary monitor in the Windows display setting (which is of course very inconvenient) or change it in the source code at [https://github.com/chrxh/alien/blob/develop/source/Gui/DisplaySettingsDialog.cpp#L22](https://github.com/chrxh/alien/blob/develop/source/Gui/DisplaySettingsDialog.cpp#L22) manually.

Just out of curiosity: ALIEN v3.0.0 is very old and many important changes have been introduced in the meantime. Is there a particular reason why you are using the old version?
